packages/components/app/styles/components/side-nav/content.scss

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// SIDE-NAV > CONTENT (PORTALS + LISTS OF ITEMS/LINKS)
+//
+
+@use "../../mixins/focus-ring" as *;
+
+
+// PANELS (wrappers use in conjunction with the portal elements_
+
+.hds-side-nav__content {
+  // we use this trick (increasing the container size here, and reducing it at single panel level)
+  // to have the panels width match the sidebar extended width (it's used in the animated sliding of the panels)
+  margin: 0 calc(var(--token-side-nav-wrapper-padding-horizontal) * -1);
+}
+
+.hds-side-nav__content-panels {
+  // see https://codepen.io/didoo/pen/YzOeRPr
```

Comment:
Just wondered if it's a good idea to link a personal Codepen here? Perhaps one of the designers could create a diagram image and add it to the documentation and that could be linked to? Or some other solution if that doesn't make sense?

Reply:
an image would not work, it's about the implementation in code (using a "trick" based on flexbox + a grid with items that are 100% in width)
the codepen is also public, so I can't think of any alternative that would achieve the same (a gist on github would not be rendered)
if you think is an issue, I can expand on the comment and explain what I'm doing below, and in this way the link is less prominent (but I would leave it anyway for future reference)